source/fundamentals/collations.txt

```diff
@@ -0,0 +1,539 @@
+==========
+Collations
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+This guide shows you how to use **collations** with MongoDB to order your
+query or aggregation operation results by string values. A collation is a set
+of character ordering and matching rules that apply to a specific language and
+locale.
+
+You can learn more about collations in the following sections in this guide:
+
+- :ref:`Collations in MongoDB <collations-mongodb>`
+- :ref:`How to Specify Collations <collations-specify>`
+- :ref:`Collation Options <collations-options>`
+- :ref:`Collation Code Examples <collations-examples>`
+
+.. _collations-mongodb:
+
+Collations in MongoDB
+---------------------
+
+MongoDB sorts strings using **binary collation** by default. The binary
+collation uses the :wikipedia:`ASCII standard <ASCII>` character values to
+compare and order strings. Certain languages and locales have specific
+character ordering conventions that differ from the ASCII character values.
+
+For example, in Canadian French, the right-most accented character
+(diacritic) determines the ordering for strings when all preceding characters
+are the same. Consider the following Canadian French words:
+
+- cote
+- coté
+- côte
+- côté
+
+When using binary collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+   :copyable: false
+
+   cote
+   coté
+   côte
+   côté
+
+When using the Canadian French collation, MongoDB sorts them in a different
+order as shown below:
+
+.. code-block:: none
+   :copyable: false
+
+   cote
+   côte
+   coté
+   côté
+
+
+.. _collations-specify:
+
+How to Specify Collations
+-------------------------
+
+MongoDB supports collations on most :doc:`CRUD operations </fundamentals/crud>`
+and aggregations. For a complete list of supported operations, see the
+:manual:`Operations that Support Collations </reference/collation/#operations-that-support-collation>`
+server manual page.
+
+You can specify the locale code and optional variant in the following string
+format:
+
+.. code-block:: none
+   :copyable: false
+
+   "<locale code>@collation=<variant code>"
+
+Specify a valid locale in the ``locale code`` placeholder and optionally a
+variant in the ``variant code`` placeholder. The following example specifies
```

Comment:
I'm unclear how to parse this sentence.  What does `placeholder` mean in this context?

Reply:
The placeholder is the presentation of the text between the angled braces. In this paragraph, "<locale code>" is a placeholder for locale code. Since it doesn't seem as intuitive as I assumed, what would you suggest to make this clear to the reader?

Reply:
I read it again and now it makes sense.  I just didn't read `"<locale code>@collation=<variant code>"` very carefully and didn't even notice the placeholders

Reply:
I'll remove that sentence since it repeats the sentence prior to the example. Hopefully, that helps guide the reader directly into the examples through which they can infer the format.